doc/developer_guide/faq.md

```diff
@@ -0,0 +1,88 @@
+
+## Can a Virtual Schema Write on the Remote Data Source?
+
+No. Exasol only supports push-down of `SELECT` statements.
+
+This design choice keeps the VS engine straightforward and highlights the absence of transaction management between Exasol and the data source.
+
+## What Kind of Activity can I Build Into the `CREATE` Request Implementations?
+
+Start with validating the input. Virtual Schemas are configured via properties. A typical property would be the name of the [connection database object](https://docs.exasol.com/saas/sql/create_connection.htm) which holds all the information required to find the data source and connect to it.
+
+Steps to verify _before_ establishing the first connection:
+
+1. Syntax and value ranges of all properties
+2. Existence of database objects mentioned in properties
+3. Are the objects accessible?
+4. Validity of connection details
+
+Next, connect to the data source and read its metadata, including its structure and data types.
+
+Map the source's structure to tables and columns so that Exasol can treat them as internal tables.
+
+Find equivalents for the source datatypes. When there is no exact match, pick the closest one and be prepared to convert data in the push-down request.
+
+## What Kind of Activity can I Build Into the `REFRESH` Request Implementations?
+
+Generally, the steps are very similar to a [`CREATE`](#what-kind-of-activity-can-i-build-into-the-create-request-implementations) request.
+
+The key difference is that users expect the refresh process to run without interruptions.
+
+That means that you should try to identify all property changes that do not affect the current setup of the Virtual Schema and handle them gracefully. That being said, there are not too many of those.
+
+For example, renaming the Virtual Schema object will always cause an interruption.
+
+
+## When is Snapshot Mode Allowed and When Recommended?
+
+We recommend reading the main article on [Snapshot Mode](https://docs.exasol.com/saas/database_concepts/snapshot_mode.htm) on the Exasol Doc portal first.
+
+In a nutshell, snapshot mode improves the efficiency of accessing Exasol's system tables.
+
+> To avoid delayed execution in such a scenario, you can run your queries to the system tables in snapshot mode. Snapshot mode ignores existing transaction locks on system tables and does not create new ones. All the queries to the system tables will run in read-only mode and show you the latest object version (latest committed version).
+
+In the context of the Virtual Schemas that is relevant for an Exasol-to-Exasol virtual schema only. Since snapshot mode is an Exasol feature, it won't help with reading 3rd-party metadata.
+
+But, if you write a virtual schema that connects to Exasol, you should always use snapshot mode when accessing system tables.
+
+Use a hint comment to tell Exasol that you want to run your query in snapshot mode:
```

Comment:
Do I understand correctly, 
* U1) that "running a query in snapshot mode" is actually triggered by the comment? 
* U2) That the syntax of the "comment" is not an _SQL comment_ but a kind-of Java-Comment? 

Maybe these key-enlightments should be named explicitly?